[BUG]: 启动报错

### 报告清单

- [X] 我已仔细阅读并了解上述注意事项。
- [X] 我已使用最新版本测试过，确定问题依旧存在。
- [X] 我确定在 [GitHub Issues](../issues) 中没有相同或相似的问题。
- [ ] 我愿意帮助修复此问题。（这会将您指定为此问题的受理人并自动创建修复分支）

### 影响版本

当前

### 问题描述

10-16 00:51:23 [ERROR] nonebot | Failed to import "nonebot_plugin_aidraw"
Traceback (most recent call last):
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python38\Scripts\nb.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\click\core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\click\core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\click\core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\click\core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\click\core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nb_cli\commands\main.py", line 30, in run
    run_bot(file, app)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nb_cli\handlers\deploy.py", line 20, in run_bot
    module = importlib.import_module(module_name)
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 22, in <module>
    nonebot.load_plugins("src/plugins")
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nonebot\plugin\load.py", line 39, in load_plugins
    return manager.load_all_plugins()
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nonebot\plugin\manager.py", line 172, in load_all_plugins
    return set(
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nonebot\plugin\manager.py", line 173, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.available_plugins))
> File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nonebot\plugin\manager.py", line 148, in load_plugin
    module = importlib.import_module(
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "c:\users\administrator\appdata\local\programs\python\python38\lib\site-packages\nonebot\plugin\manager.py", line 237, in exec_module
    super().exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\src\plugins\nonebot_plugin_aidraw\__init__.py", line 1, in <module>
    from . import draw
  File ".\src\plugins\nonebot_plugin_aidraw\draw.py", line 26, in <module>
    from .limit import daily_limiter, limiter
  File ".\src\plugins\nonebot_plugin_aidraw\limit.py", line 12, in <module>
    class DailyLimiter(BaseModel):
  File ".\src\plugins\nonebot_plugin_aidraw\limit.py", line 15, in DailyLimiter
    count: defaultdict[int, int] = Field(defaultdict(int), init=False)
TypeError: 'type' object is not subscriptable

### 重现步骤

_No response_

### 预期行为

_No response_

### 实际行为

_No response_

### 日志信息

_No response_

### 额外补充

_No response_

--- a/nonebot_plugin_aidraw/limit.py
+++ b/nonebot_plugin_aidraw/limit.py
@@ -12,7 +12,7 @@ from .config import daily_times
 class DailyLimiter(BaseModel):
     max: int
     day: int = Field(default=0, init=False)
-    count: defaultdict[int, int] = Field(defaultdict(int), init=False)
+    count: defaultdict = Field(defaultdict(int), init=False)
 
     def last(self, key: int) -> int:
         return self.max - self.count[key]

